src/hist/core.py

```diff
@@ -13,12 +14,11 @@ def __init__(self, *args, **kwargs):
             Initialize Hist object. Axis params must contain the names.
         """
 
+        # ToDo: why cannot use super(Histogram, self).__init__(*args, **kwargs)
         super(BaseHist, self).__init__(*args, **kwargs)
```

Comment:
Can't you use this:

```suggestion
        super().__init__(*args, **kwargs)
```

Reply:
We can use this. But I don't know why this happens.